samples/nrf9160/http_application_update/src/main.c

```diff
@@ -138,15 +140,15 @@ static int dfu_button_init(void)
 		return 1;
 	}
 	err = gpio_pin_configure(gpiob, DT_ALIAS_SW0_GPIOS_PIN,
-				 GPIO_DIR_IN | GPIO_INT | GPIO_PUD_PULL_UP |
-					 GPIO_INT_EDGE | GPIO_INT_ACTIVE_LOW);
+				 GPIO_INPUT | GPIO_PULL_UP);
```

Comment:
Instead of `GPIO_PULL_UP`, use `DT_ALIAS_SW0_GPIOS_FLAGS` here, and then configure the interrupt with `GPIO_INT_EDGE_TO_ACTIVE`.

Reply:
will do.